lib/index.js

```diff
@@ -47,10 +47,10 @@ function NwBuilder(options) {
         argv: process.argv.slice(2)
     };
     // Intercept the platforms and check for the legacy platforms of 'osx' and 'win' and
-    // replace with 'osx32', 'osx64', and 'win32', 'win64' respectively.
+    // replace with 'osx64', and 'win32', 'win64' respectively.
     if(typeof options.platforms != 'undefined'){
         if(options.platforms.indexOf('osx') >= 0){
-            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx32', 'osx64');
+            options.platforms.splice(options.platforms.indexOf('osx'), 1, 'osx64');
```

Comment:
Ugh, you're right about it all being a breaking change :/

I love merging stuff from new contributors but maybe we should wait. We can both keep an eye out for more breaking changes so we can merge this ASAP 😛